job_dsl/dep_update.groovy

```diff
@@ -3,56 +3,70 @@
 // Alternative is to use parameters with JJB vars as the defaults.
 library "rpc-gating@${RPC_GATING_BRANCH}"
 common.shared_slave(){
```

Comment:
Why not include this in the try block?

Reply:
shared_slave(), internal_slave(), etc. all call use_node() which does its own error handling, including cleanup.  The cleanup will remove the venv which we need to create the JIRA issue. We could update the catch to pull down the venv if it doesn't exist, but that's just adding potentially more and more time to job's execution time.